packages/doctype-tile/src/tile-doctype.ts

```diff
@@ -255,4 +180,41 @@ export class TileDoctype<T = Record<string, any>> extends Doctype {
         return _signDagJWS(commit, did, this.controllers[0])
     }
 
+    /**
+     * Create genesis commit.
+     * Placed on the TileDoctype class as a static method only for use in testing.
+     * @param did - DID making (and signing) the commit
+     * @param content - genesis content
+     * @param metadata - genesis metadata
+     * @private
+     */
+    static async _makeGenesis<T>(did: DID, content: T | null, metadata?: TileMetadataArgs): Promise<CeramicCommit> {
```

Comment:
Any reason to keep this @private? Seems like it could be a useful function in general.

Reply:
Yeah that makes sense.  Updated